Documentation/network/standalone.rst

```diff
@@ -0,0 +1,1610 @@
+     --bpf-lb-acceleration=native \\
+     --devices=bond0
+
+The Maglev selection consumes significantly more memory due to the needed lookup tables.
+If the use case for Cilium Standalone Gateway is to just act as a proxy for translating
+from one service VIP to another service VIP (e.g. IPv4 to IPv6 one) such that per service
+only one backend is required, then sticking with the Random mode (default) is sufficient.
+
+Introspecting the raw Maglev lookup tables from BPF side can be achieved through
+``docker exec cilium-gateway cilium bpf lb maglev list``.
+
+Backend State Management
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+For maintenance, quarantining or other purposes it can be necessary to drain traffic
+from a given backend. In such case, the load-balancer will not consider those backends
+for traffic forwarding, meaning, they are excluded for new connections. Ongoing connections
+are still kept intact until a backend is removed from the given service entirely. Once
+the backend is removed from the service, then (still) ongoing traffic will be dropped.
+
+The backend state is presented in the service dump, and can be one of ``active`` (default),
+``terminating``, ``quarantined``, ``maintenance``:
+
+.. parsed-literal::
+
+  docker exec cilium-gateway cilium service list
+  ID   Frontend            Service Type   Backend
+  1    86.109.5.207:8080   ExternalIPs    1 => 1.1.1.1:80 (active)
+                                          2 => 1.1.1.2:80 (active)
+
+Semantically the three states ``terminating``, ``quarantined`` and ``maintenance`` are
+the same and all of them exclude the provided backend for new connections. However, third
+party software built on top of this framework may use them for different purposes:
+
+- ``quarantined``: An out-of-band health checking mechanism determined that the backend
+  was flaky, and therefore briefly puts the backend out of service.
+- ``maintenance``: The backend is taken out of service for maintenance purpose such as
+  for updating the backend software.
+- ``terminating``: The backend is taken out of service indefinitely.
+
+States can transition from:
+
+- ``active`` into ``terminating``, ``quarantined`` or ``maintenance``
+- ``quarantined`` into ``active`` or ``terminating``
+- ``maintenance`` into ``active``
+- ``terminating`` is a final state
+
+The above backend state management is supported for both Random and Maglev backend selection.
+
+The state for a given backend can be updated as follows:
+
+.. parsed-literal::
+
+  docker exec cilium-gateway cilium service update --backends 1.1.1.2:80 --states maintenance
+  Updating backend states
+  Updated service with 1 backends
+
+  docker exec cilium-gateway cilium service list
+  ID   Frontend            Service Type   Backend
+  1    86.109.5.207:8080   ExternalIPs    1 => 1.1.1.1:80 (active)
+                                          2 => 1.1.1.2:80 (maintenance)
+
+The backend state is global, meaning, if a backend IP:port is part of multiple services,
```

Comment:
"global" can have implications around multi-cluster, do we want to use that terminology here? (I realise standalone gateway is a bit of a different beast, but I figure I should ask in case it does overlap / confuse).

Reply:
changed into:

The backend state is global for the load-balancer node, meaning, if a backend IP:port is part
of multiple services, then all of them are updated accordingly: